src/PhaseDiagrams/PhaseDiagrams.jl

```diff
@@ -32,6 +32,13 @@ struct PhaseDiagram_LookupTable <: AbstractPhaseDiagramsStruct
     Vs::Any
     VpVs::Any
     cpxFrac::Any
+    solid_Vp::Any
```

Comment:
This is `struct` is not parametric, which may result in type instabilities.  I would make it parametric with an inner constructor which promotes all the variables that are currently annotated as `Any` to the same type. Something like this:
```julia
struct PhaseDiagram_LookupTable{S,T} <: AbstractPhaseDiagramsStruct
    Type::S
    Name::S
    HeaderText::T
    rockRho::T
    meltRho::T
    etc::T
    function PhaseDiagram_LookupTable(Type, HeaderText, Name, rockRho, meltRho, etc)
        args = promote(HeaderText, rockRho, meltRho, etc)
        T = eltype(args)
        new{String, T}(Type, Name, args...)
    end
end
```